k8s/dependencies/pelias/api.yaml

```diff
@@ -40,6 +40,12 @@ spec:
             - name: PELIAS_CONFIG
               value: "/etc/config/pelias.json"
           resources:
+<<<<<<< HEAD
```

Comment:
This shouldn't be here!

Reply:
no! I know ;)
I'm trying to remove this pull request and start over.